### ISSUE TITLE
Quest: Blueleaf tuber

### ISSUE THREAD

**Issue by thestranger064** [dalaranwow/dalaran-wow#4381]
http://wotlk.openwow.com/quest=1221

http://wotlk.openwow.com/item=5880 
http://www.aowow.org/?item=5880 Cannot get it to work I been running all over the instance and clicked, even the escort area for the willix quest, and nothing happens.

According to some earlier reports on this, this is supposed to be fixed, even with a auto complete feature.
Sadly this is not the case. Currently it is not working.

I am also playing aliance.

[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
DB/Quest: Blueleaf tuber - Insert script for Snufflenose Gopher

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4381
* commit: e09b55e343a07b91754a1b2f02af7835e405bb2a

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by thestranger064** [comment]
I understand, thank you replying.

**Comment by TeridanDW** [comment]
Hello, something that might be an issue:

> Also i would like to add that npc http://www.wowhead.com/npc=4781/snufflenose-gopher summoned by spell http://www.wowhead.com/spell=6918/summon-snufflenose needs to be `set reach on passive` on summon. Currently he is aggressive and if player is high level (creature scales with the owner) he will engage creatures from very far away.

**Comment by TeridanDW** [comment]
After a fast test on PTR:
 - The original issue persist: Item https://wowgaming.altervista.org/aowow/?item=5880 can't be used anywhere in the dungeon.

 - Issue:

> Also i would like to add that npc http://www.wowhead.com/npc=4781/snufflenose-gopher summoned by spell http://www.wowhead.com/spell=6918/summon-snufflenose needs to be set reach on passive on summon. Currently he is aggressive and if player is high level (creature scales with the owner) he will engage creatures from very far away.

can not be tested because of issue 1.

**Comment by Lothloryen** [comment]
Hello there,

It will be corrected with the next realm maintenance. Thank you for the reports.

**Comment by wrathofthebug** [comment]
It's been almost 9 months with multiple maintenance past. Quest still not working as intended.
